src/Models/Constants/ActionTypes.ts

```diff
@@ -25,3 +25,28 @@ export const SET_ADT_SCENE_BUILDER_SELECTED_ELEMENT =
 export const SET_ADT_SCENE_ELEMENT_SELECTED_OBJECT_IDS =
     'SET_ADT_SCENE_ELEMENT_SELECTED_OBJECT_IDS';
 export const SET_ADT_SCENE_BUILDER_MODE = 'SET_ADT_SCENE_BUILDER_MODE';
+export const SET_ADT_SCENE_PAGE_MODE = 'SET_ADT_SCENE_PAGE_MODE';
```

Comment:
do you need the scene page mode? or is that a merge mistake? Doesnt' seem related to OAT.

Reply:
yeah this might be a merge miss, i think we can pull it if its an unused reference